crates/sozo/ops/src/statistics.rs

```diff
@@ -0,0 +1,187 @@
+use std::fs::{self, File};
+use std::io;
+use std::path::PathBuf;
+
+use anyhow::Result;
+use camino::Utf8PathBuf;
+use starknet::core::types::contract::SierraClass;
+use starknet::core::types::FlattenedSierraClass;
+
+#[derive(Debug, PartialEq)]
+pub struct ContractStatistics {
+    pub contract_name: String,
+    pub number_felts: u64,
+    pub file_size: u64,
+}
+
+fn read_sierra_json_program(file: &File) -> Result<FlattenedSierraClass> {
+    let contract_artifact: SierraClass = serde_json::from_reader(file)?;
```

Comment:
sierra file can be pretty huge, just a small perf nit

```suggestion
    let contract_artifact: SierraClass = serde_json::from_reader(BufReader::new(file))?;
```